### PR TITLE
add `ci` directory

### PR DESCRIPTION
**Pull request by justinpjose**
### What
**Describe what you have changed and why.**
- add `ci` directory

### How to review
**Describe the steps required to test the changes.**
- Check if the code changes make sense

### Who can review
**Describe who worked on the changes, so that other people can review.**
- Anyone

### CODE REVIEW

File: ci/audit.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  source:
+    repository: onsdigital/dp-concourse-tools-nancy
+    tag: latest
+
+inputs:
+  - name: dp-cache
+    path: dp-cache
+
+run:
+  path: dp-cache/ci/scripts/audit.sh
```
Comment (cookel2) [inline ci/audit.yml:15]: new line needed

**Review by cookel2 — CHANGES_REQUESTED** [review]
Some of the files are missing new lines at the end.

File: ci/audit.yml (hunk 1 above)
Comment (justinpjose) [reply ci/audit.yml]: Done

File: ci/scripts/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-cache
+  make build
+popd
```
[the remark this answers is not in the document]
Comment (justinpjose) [reply ci/scripts/build.sh]: Done

File: ci/scripts/unit.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -eux
+
+cwd=$(pwd)
+
+pushd $cwd/dp-cache
+  make test
+popd
```
[the remark this answers is not in the document]
Comment (justinpjose) [reply ci/scripts/unit.sh]: Done

**Review by cookel2 — APPROVED** [review]
looks good to me

**Pull request merged**